php/index.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+header("Access-Control-Allow-Origin: *");
+
+$pdo = new PDO("mysql:dbname=coucou;host=127.0.0.1", "root", "");
```

Comment:
Tu pourrais récupérer les données de connexion à partir de l'environnement (voir si l'hébergement mutualisé permet de gérer les variables d'environnement) :)

Reply:
@WilliamTraoreee do you think you'll be able to make these small changes?

Reply:
Totally !

Reply:
Wait, make sure that your hosting environment allows usage of env vars ! You can also use a function like env('VARNAME', default value) if env() is disabled